bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/help/SkontoHelpFragment.kt

```diff
@@ -0,0 +1,390 @@
+import net.gini.android.capture.ui.theme.modifier.tabletMaxWidth
+import net.gini.android.capture.view.InjectedViewAdapterInstance
+
+
+class SkontoHelpFragment : Fragment() {
+
+    private val isBottomNavigationBarEnabled =
+        GiniCapture.getInstance().isBottomNavigationBarEnabled
+
+    private val customBottomNavBarAdapter: InjectedViewAdapterInstance<SkontoHelpNavigationBarBottomAdapter>? =
+        GiniBank.skontoHelpNavigationBarBottomAdapterInstance
+    private var customBottomNavigationBarView: View? = null
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (GiniCapture.hasInstance() && !GiniCapture.getInstance().allowScreenshots) {
+            requireActivity().window.disallowScreenshots()
+        }
+        ActivityHelper.forcePortraitOrientationOnPhones(activity)
+
+        if (resources.getBoolean(net.gini.android.capture.R.bool.gc_is_tablet)) {
+            requireActivity().window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_NOTHING)
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
+    ): View {
+        customBottomNavigationBarView =
+            container?.let { customBottomNavBarAdapter?.viewAdapter?.onCreateView(it) }
+
+        return ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            setContent {
+                GiniTheme {
+                    ScreenContent(
+                        isBottomNavigationBarEnabled = isBottomNavigationBarEnabled,
+                        customBottomNavBarAdapter = customBottomNavBarAdapter,
+                        navigateBack = {
+                            findNavController()
+                                .navigateUp()
+                        }
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+@Composable
+private fun ScreenContentPreview() {
+    GiniTheme {
+        ScreenContent(
+            navigateBack = {},
+            screenColorScheme = SkontoHelpScreenColors.colors(),
+            isBottomNavigationBarEnabled = false,
+            customBottomNavBarAdapter = null
+        )
+    }
+}
+
+@Preview
+@Composable
+private fun ScreenContentPreviewLight() {
+    GiniTheme {
+        ScreenContent(
+            navigateBack = {},
+            screenColorScheme = SkontoHelpScreenColors.colors(),
+            isBottomNavigationBarEnabled = false,
+            customBottomNavBarAdapter = null
+        )
+    }
+
+}
+
+@Composable
+private fun ScreenContent(
+    navigateBack: () -> Unit,
+    modifier: Modifier = Modifier,
+    screenColorScheme: SkontoHelpScreenColors = SkontoHelpScreenColors.colors(),
+    isBottomNavigationBarEnabled: Boolean,
```

Comment:
[Requirement]
By kotlin convention the required parameters should placed before optional